cli/admin/README.md

```diff
@@ -0,0 +1,130 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Pravega Admin CLI
+
+Pravega Admin CLI is a tool for inspecting and exploring Pravega deployments.
+
+## Prerequisites
+
+- Java 8+
```

Comment:
This should be 11 now. Pretty soon we'll begin using JDK11 APIs so we can't really claim Java8 compatibility.

Reply:
The Java version change has been done while the PR was open. I have changed it.